graphene/contrib/django/fetcher.py

```diff
@@ -0,0 +1,28 @@
+from functools import wraps
+from graphql.core.utils.get_field_def import get_field_def
+from graphql.core.type.definition import GraphQLList, GraphQLNonNull
+
+
+def get_fields(info):
+    field_asts = info.field_asts[0].selection_set.selections
+    only_args = []
+    _type = info.return_type
+    if isinstance(_type, (GraphQLList, GraphQLNonNull)):
+        _type = _type.of_type
+
+    for field in field_asts:
+        field_def = get_field_def(info.schema, _type, field)
+        f = field_def.resolver
+        fetch_field = getattr(f, 'django_fetch_field', None)
+        if not fetch_field:
+            continue
+        only_args.append(fetch_field.attname)
+    return only_args
```

Comment:
This function seems useful outside Django, to get the fields from an info object. Is there any way this could be refactored into  graphene core? I'm willing to do the work, but I'm not familiar enough with the codebase to know where this should fit.